### PR TITLE
New Bootndoes

### PR DESCRIPTION
**Pull request by AurevoirXavier**

### CODE REVIEW

File: node/service/res/crab/crab.json [hunk 1]
```diff
@@ -3,8 +3,8 @@
 	"id": "crab",
 	"chainType": "Live",
 	"bootNodes": [
-		"/dns/g1.p2p.crab.darwinia.network/tcp/30333/p2p/12D3KooWFqHZkyv6iabxxqiHdNjWb4c7EfmBqMNCyqLCCVZm8yyQ",
-		"/dns/g2.p2p.crab.darwinia.network/tcp/30333/p2p/12D3KooWPiza2NAD6CjdBGtfUd3pfDnZXysYKzumejGHafW3Y8xP"
+		"/dns/g1.crab-p2p.darwinia.network/tcp/30333/p2p/12D3KooWFqHZkyv6iabxxqiHdNjWb4c7EfmBqMNCyqLCCVZm8yyQ",
+		"/dns/g2.crab-p2p.darwinia.network/tcp/30333/p2p/12D3KooWPiza2NAD6CjdBGtfUd3pfDnZXysYKzumejGHafW3Y8xP"
```
Comment (AsceticBear) [inline node/service/res/crab/crab.json:7]: Why update those addresses? The old node is no longer maintained?

Comment (AurevoirXavier) [reply node/service/res/crab/crab.json:7]: Old domain was dropped.

**Comment by HackFisher** [comment]
@wi1dcard please help review.

**Review by wi1dcard — APPROVED** [review]

**Comment by wi1dcard** [comment]
> @wi1dcard please help review.

LGTM

**Review by AsceticBear — APPROVED** [review]

**Pull request merged**